docs/sources/visualizations/geomap.md

```diff
@@ -12,6 +12,104 @@ The Geomap panel visualization allows you to view and customize the world map us
+
+There are four layer types to choose from in the Geomap visualization.
+
+* **Marker** renders a marker at each data point.
+* **Heatmap** visualizes a heatmap of the data.
+* **GeoJSON** renders static data from a geojson file.
+* **Icon at last point (alpha)** renders an icon at the last point in a series of points.
+
+### Layer Controls
+
+The layer controls allow you to create layers, change their name, reorder and delete layers.
+
+* **Add layer** creates an additional, configurable data layer for the Geomap visualization. When you add a layer you will be prompted to select a layer type. The layer type can be changed at any point during panel configuration. See the **Layer Types** section above for details on each layer type.
+* The layer controls allow you to rename, delete, and reorder the layers of the panel.
+
+You can add multiple layers of data to a single Geomap panel in order to create rich, detailed visualizations.
+
+### Location
+
+The Geomap panel needs a source of geographical data. This data comes from a database query, and there are four mapping options for your data.
+
+- **Auto** automatically searches for location data. Use this option when your query is based on one of the following names for data fields.
+  - geohash: “geohash”
+  - latitude: “latitude”, “lat”
+  - longitude: “longitude”, “lng”, “lon”
+  - lookup: “lookup”
+- **Coords** specifies that your query holds coordinate data. You will get prompted to select numeric data fields for latitude and longitude from your database query.
+- **Geohash** specifies that your query holds geohash data. You will get prompted to select a string data field for the geohash from your database query.
+- **Lookup** specifies that your query holds location name data that needs to be mapped to a value. You will get prompted to select the lookup field from your database query and a gazetteer. The gazetteer is the directory that is used to map your queried data to a geographical point.
+
+### Markers layer
+
+The markers layer allows you to display data points as different marker shapes such as circle, squares, triangles, stars, and more.
+
+![Markers Layer](/static/img/docs/geomap-panel/geomap-markers-8-1-0.png)
+
+![Markers Layer Options](/static/img/docs/geomap-panel/geomap-markers-options-8-1-0.png)
+
+- **Marker Color** configures the color of the marker. The default `Fixed size` keeps all points a single color. There is an alternate option to have multiple colors depending on the data point values and the threshold set at the `Thresholds` section.
+- **Marker Size** configures the size of the marker. Default is `Fixed size`, making all marker size the same regardless of the data points. However, there is also an option to scale the circles to the corresponding data points. `Min` and `Max` marker size has to be set such that the Marker layer can scale within these range.
+- **Marker Shape** provides you with the flexibility to visualize the data points differently.
```

Comment:
I think we now support a lot more options (including all unicons), so we may want to mention increased options at very least here

Reply:
Great call out. I'm going to resolve this and push another change with an expanded markers section.